packages/accounts-2fa/2fa-server.js

```diff
@@ -0,0 +1,123 @@
+import { Accounts } from 'meteor/accounts-base';
+import twofactor from 'node-2fa';
+import QRCode from 'qrcode-svg';
+import { Meteor } from 'meteor/meteor';
+
+Accounts.is2faEnabledForUser = selector => {
```

Comment:
Why is this method exposed in the Accounts object?

Reply:
Because I also need to call it inside the accounts-password 
https://github.com/meteor/meteor/blob/426872ebb4fe00c14914a66f5f4705d3d88823be/packages/accounts-password/password_server.js#L187-L190

Reply:
Same as above, we should start with `_`

Reply:
Isn't this necessary if I want to show a screen saying that this user has 2fa enabled or not?

Reply:
This is the server part @renanccastro. The client method it's exposed [already](https://github.com/meteor/meteor/blob/65fb19a105d769c836bcd9bbb40b67006f2798b1/packages/accounts-2fa/2fa-client.js#L12-L21).

Reply:
Done